src/support.js

```diff
@@ -71,25 +79,39 @@ const supportCommands = () => {
   Cypress.Commands.add('clickToolbarButton', clickToolbarButton)
 
 
-  // Check for notices and warnings
+  /**
+   * Check for notices and warnings
+   *
+   * @memberof cy
+   * @method checkForPhpNoticesOrWarnings
+   * @returns Chainable
+   */
   const checkForPhpNoticesOrWarnings = () => {
     cy.log('**Check for PHP notices and warnings**')
 
-    // cy.contains('Notice:').should('not.exists')
```

Comment:
When I initially converted this over, I had the issue that we did have pages where we had the text string `Warning:` in the output as legitimate content and that triggered this check. Did you check for this? Can we secure this further somehow?

Reply:
Hmm... this would only work for me, I guess. Good point! I changed the code to detect bold Warnings/... only.